### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by QuantumPhi** [izuzak/atom-pdf-view#45]
Atom deprecation cop complains on use of `atom.workspaceView`, expects atom.views.getView(atom.workspace)`.

**Comment by izuzak** [comment]
Which version of the package are you using, @QuantumPhi? This should be fixed as of https://github.com/izuzak/atom-pdf-view/pull/44. There are no workspaceView references currently:

https://github.com/izuzak/atom-pdf-view/search?utf8=%E2%9C%93&q=workspaceView

**Comment by QuantumPhi** [comment]
I just noticed I was running v0.14.0. Thanks.

**Comment by QuantumPhi** [comment]
Side note: running `apm install pdf-view` from command line installs v0.14.0, while installing through Atom GUI installs v0.15.0.

**Comment by izuzak** [comment]
Well that's strange. I'm seeing the same behavior:

```
$ apm list --installed
/Users/izuzak/.atom/packages (1)
└── the-closer@0.3.0

$ apm install pdf-view
Installing pdf-view to /Users/izuzak/.atom/packages ✓

$ apm list --installed
/Users/izuzak/.atom/packages (2)
├── pdf-view@0.14.0
└── the-closer@0.3.0
```

However, the latest published version is `0.15.0`:

```
$ apm show pdf-view
pdf-view
├── 0.15.0
├── https://github.com/izuzak/atom-pdf-view
├── Atom PDF viewer based on PDF.js
├── 3345 downloads
└── 28 stars

Run `apm install pdf-view` to install this package.
```

And the Settings View show that 0.15.0 is the latest version and this is the version which is installed if I install using Settings View:

![screen shot 2015-01-30 at 15 07 13](https://cloud.githubusercontent.com/assets/38924/5976763/d827e7e8-a891-11e4-9e74-fb6c6e7713fe.png)

@thedaniel @kevinsawicki Do you perhaps know what might be causing this difference?

**Comment by izuzak** [comment]
@QuantumPhi this should be fixed now, see https://github.com/atom/atom/pull/5417 for the :newspaper:. Thanks for reporting this.
